File: SDKs/velopayments.com/2.26.127/go/pkg/models/shared/payeedelta.go
package shared



type PayeeDelta struct {
    DbaName *string `json:"dbaName,omitempty"`
    DisplayName *string `json:"displayName,omitempty"`
    Email *string `json:"email,omitempty"`
    OnboardedStatus *OnboardedStatus2Enum `json:"onboardedStatus,omitempty"`
    PayeeCountry *string `json:"payeeCountry,omitempty"`
    PayeeID string `json:"payeeId"`
    RemoteID string `json:"remoteId"`
    
}

File: SDKs/velopayments.com/2.26.127/typescript/src/sdk/models/shared/payeedelta.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { OnboardedStatus2Enum } from "./onboardedstatus2enum";



export class PayeeDelta extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=dbaName" })
  dbaName?: string;

  @SpeakeasyMetadata({ data: "json, name=displayName" })
  displayName?: string;

  @SpeakeasyMetadata({ data: "json, name=email" })
  email?: string;

  @SpeakeasyMetadata({ data: "json, name=onboardedStatus" })
  onboardedStatus?: OnboardedStatus2Enum;

  @SpeakeasyMetadata({ data: "json, name=payeeCountry" })
  payeeCountry?: string;

  @SpeakeasyMetadata({ data: "json, name=payeeId" })
  payeeId: string;

  @SpeakeasyMetadata({ data: "json, name=remoteId" })
  remoteId: string;
}
